src/main/java/org/fundacionjala/pivotal/core/e2e/CommonMethods.java

```diff
@@ -23,7 +23,7 @@ private CommonMethods() {
      * @param text       String.
      */
     public static void setWebElement(final WebElement webElement, final String text) {
-        waitWebElement(webElement).clear();
+        waitWebElement(webElement);
```

Comment:
Are you sure to do this ?

Reply:
for now yes, we're gonna create other method in order to support comboboxes